pages/linux/ffuf.md

```diff
@@ -0,0 +1,28 @@
+# ffuf
+
+> Subdomain and directory discovery tool.
+> More information: <https://github.com/ffuf/ffuf>.
+
+- Discover directories using a [w]ordlist on a target [u]rl with [c]olorized and [v]erbose ouput:
+
+`ffuf -w {{path/to/wordlist}} -u {{https://target/FUZZ}} -c -v`
+
+- Fuzz [H]ost-headers with a host file on a traget website and [m]atch HTTP 200 [c]ode responses:
```

Comment:
Do we like all of the [H] in these?

I understand they signify the flags, but does it have long flags. 

PS: sorry if I have already asked this

Reply:
The ` [H] ` should be swapped as the flag is for "header" configuration and not "host" configuration.  
Good catch.

There are longer flags, but there are no instances of double tac(`--`) flags.
So there is `-H` and `-maxtime`.  There are no instances of `--Header` or `--MaximumTime`

addresed in 56d52cd8695b7ba70e0884664dff62a436c842c0